less/components/btn.less

```diff
@@ -1,4 +1,5 @@
 .btn {
+  white-space: normal;
```

Comment:
That's a [good point](https://github.com/mozilla/teach.webmaker.org/pull/943/files#r31200930), do you think you can actually add that as a comment in the code?